src/app/public/modules/split-view/split-view-workspace.component.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  HostListener,
+  Input,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+
+import {
+  SkyCoreAdapterService,
+  SkyMediaQueryService
+} from '@skyux/core';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  SkySplitViewMediaQueryService
+} from './split-view-media-query.service';
+
+import {
+  SkySplitViewService
+} from './split-view.service';
+
+@Component({
+  selector: 'sky-split-view-workspace',
+  templateUrl: 'split-view-workspace.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    SkySplitViewMediaQueryService,
+    { provide: SkyMediaQueryService, useExisting: SkySplitViewMediaQueryService }
+  ]
+})
+export class SkySplitViewWorkspaceComponent implements OnDestroy, OnInit {
+  public set isMobile(value: boolean) {
```

Comment:
Return type. Also could we add a line before this?

Reply:
Setters don't have return types. I added a new line.

Reply:
Oh. I forgot that it would yell at you for that. Ok.